src/qml/Tracking.qml

```diff
@@ -39,12 +39,16 @@ Item{
               if( ( geometryType === QgsWkbTypes.LineGeometry && vertexCount == 3 ) ||
                   ( geometryType === QgsWkbTypes.PolygonGeometry && vertexCount == 4 ) )
               {
-                  featureModel.create()
+                  if ( ! featureModel.create() ) {
+                      displayToast( qsTr( "Failed to create feature!" ) )
+                  }
                   mainModel.feature = featureModel.feature
               }
               else
               {
-                  featureModel.save()
+                  if ( ! featureModel.save() ) {
+                      displayToast( qsTr( "Failed to save feature!" ) )
```

Comment:
Log message instead of toast. Otherwise this would pop up all the time during the tracking.